actions/user-delete

```diff
@@ -0,0 +1 @@
+user-create
```

Comment:
Bike shedding: Since you're symlinking anyway, you could make `user-create` a symlink as well and change the implementation filename to `manage_users.py` so that the syntax highlighting would work on the webview (plus it's a tiny bit more accurate, since the file contains more than just the `user_create` function).